docker-compose.yml

```diff
@@ -0,0 +1,35 @@
+version: "3.9"
+
+services:
+  backend:
+    build: .
+    depends_on:
+      - db
+    ports:
+      - 8080:8080
+    environment:
+      - CB_HOST
+      - CB_USER
+      - CB_PSWD
+      - COLUMNS
```

Comment:
Don't think we even need these env variables now, since we have `mix-and-match.yml` for running with different configurations.

Reply:
ooo, good catch.